src/CorePluginAPI/Game/Skills/ESkillFlag.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace QuantumCore.API.Game.Skills;
+
+public enum ESkillFlag
```

Comment:
Shouldn't this have `[Flags]` annotation? Also please add enum member values for easier reading (Rider can do this for u)

Reply:
Yes you're correct. Updated the code to reflect that.

My question is regarding the enum member values, is that something like the following ?

```csharp
[Flags]
public enum ESkillFlag
{
    [EnumMember(Value = "ATTACK")] Attack = 1 << 0,
```

Reply:
Why do you need `EnumMemberAttribute` here?